interpreter/binary/decode.ml

```diff
@@ -555,6 +556,7 @@ let import_desc s =
   | 0x01 -> TableImport (table_type s)
   | 0x02 -> MemoryImport (memory_type s)
   | 0x03 -> GlobalImport (global_type s)
+  | 0x04 -> ignore (vu32 s); EventImport (at var s)
```

Comment:
You need to check that the value is 0, otherwise it won't be forward-compatible. Also, a single zero byte is enough for an unused extension hook (since it's upward-compatible to LEB), and is what we did elsewhere.

In similar other places, the decoder uses the `zero` function for this.

Moreover, I think you want to introduce an `event_type` decoding function for this, since the same it's reused for event definitions below. Something like:
```
let event_type s =
  zero s;
  let x = at var s in
  EventType x
```

Reply:
Done